python/Ganga/GPIDev/Lib/Registry/BoxRegistry.py

```diff
@@ -155,14 +161,11 @@ def proxy_remove(self, obj_id):
 
         self._remove(self._get_obj(obj_id))
 
+    def getSlice(self):
+        return self.stored_slice
+
     def getProxy(self):
-        slice = BoxRegistrySlice(self.name)
-        slice.objects = self
-        proxy = BoxRegistrySliceProxy(slice)
-        proxy.add = self.proxy_add
-        proxy.rename = self.proxy_rename
-        proxy.remove = self.proxy_remove
-        return proxy
+        return self.stored_proxy
```

Comment:
This is only because I've re-written it, I'm not about to go in and change the design as the getters return different things (or don't) for certain repos anyway. This is better in a future PR imo.